discouraged

```diff
@@ -4926,6 +4926,8 @@
 "dfhnorm2" is used by "hilnormi".
 "dfhnorm2" is used by "normf".
 "dfhnorm2" is used by "normval".
+"dfid2" is used by "fsplit".
```

Comment:
I think ~dfid2 should be replaced by ~df-id  in the proof of ~fsplit, making it 1 byte longer...

Reply:
It looks like the modification of ~fsplit to use df-id instead of dfid2 would not be too complicated, but I don't see such a modification as "1 byte longer" ? Maybe you can do it in a following PR ?
At least, I don't see this as merge-blocking.

Reply:
(I did `MM-PA> EXPAND dfid2`)

Reply:
This is what I did:

MM> pr fsplit
Entering the Proof Assistant.  HELP PROOF_ASSISTANT for help, EXIT to exit.
You will be working on statement (from "SHOW STATEMENT fsplit"):
25840 fsplit $p |- `' ( 1st |` _I ) = ( x e. _V |-> <. x , x >. ) $= ... $.
Note:  The proof you are starting with is already complete.
MM-PA> sh n/lem
 88 vex          $p |- x e. _V
 90 vex          $p |- y e. _V
 91 88,90 brcnv  $p |- ( x `' ( 1st |` _I ) y <-> y ( 1st |` _I ) x )
126 vex          $p |- x e. _V
127 126 brres    $p |- ( y ( 1st |` _I ) x <-> ( y 1st x /\ y e. _I ) )
242 19.42v       $p |- ( E. z ( ( 1st ` y ) = x /\ y = <. z , z >. ) <-> ( (
                                      1st ` y ) = x /\ E. z y = <. z , z >. ) )
317 vex          $p |- z e. _V
319 vex          $p |- z e. _V
320 317,319 op1std  $p |- ( y = <. z , z >. -> ( 1st ` y ) = z )
321 320 eqeq1d   $p |- ( y = <. z , z >. -> ( ( 1st ` y ) = x <-> z = x ) )
322 321 pm5.32ri  $p |- ( ( ( 1st ` y ) = x /\ y = <. z , z >. ) <-> ( z = x /\
                                                            y = <. z , z >. ) )
323 322 exbii    $p |- ( E. z ( ( 1st ` y ) = x /\ y = <. z , z >. ) <-> E. z (
                                                   z = x /\ y = <. z , z >. ) )
379 fo1st        $p |- 1st : _V -onto-> _V
383 fofn         $p |- ( 1st : _V -onto-> _V -> 1st Fn _V )
384 379,383 ax-mp  $a |- 1st Fn _V
386 vex          $p |- y e. _V
393 fnbrfvb      $p |- ( ( 1st Fn _V /\ y e. _V ) -> ( ( 1st ` y ) = x <-> y
                                                                      1st x ) )
394 384,386,393 mp2an  $p |- ( ( 1st ` y ) = x <-> y 1st x )
432 dfid2        $p |- _I = { <. z , z >. | z = z }
Press <return> for more, Q <return> to quit, S <return> to scroll to end... q
MM-PA> del step 432
A 2-step subproof at step 432 was deleted.  Steps 432:689 are now 431:688.
431                      eleq1i.1=?        $? |- _I = { <. z , z >. | z = z }
MM-PA> ass 431 df-id
MM-PA> improve all
A proof of length 1 was found for step 432.
A proof of length 1 was found for step 431.
Steps 431 and above have been renumbered.
CONGRATULATIONS!  The proof is complete.  Use SAVE NEW_PROOF to save it.
Note:  The Proof Assistant does not detect $d violations.  After saving
the proof, you should verify it with VERIFY PROOF.
MM-PA> sh n/c
Proof of "fsplit":
---------Clip out the proof below this line to put it in the source file:
      ( vy vz cv c1st cid cres ccnv wbr copab cop wceq cvv cmpt vex wcel wa wex
      exbii bitri brcnv brres cfv 19.42v op1std eqeq1d pm5.32ri wfn wb wfo fofn
      fo1st ax-mp fnbrfvb mp2an df-id eleq2i nfe1 elopab biantru 3bitr4i bitr2i
      19.9 equid anbi12i 3bitr3ri id opeq12d eqeq2d ceqsexv opabbii wrel relcnv
      dfrel4v mpbi mptv 3eqtr4i ) ADZBDZEFGZHZIZABJZVSVRVRKZLZABJWAAMWDNWBWEABW
      BVSVRVTIZWEVRVSVTAOZBOZUAWFVSVREIZVSFPZQZWEVSVREFWGUBWKCDZVRLZVSWLWLKZLZQ
      ZCRZWEVSEUCZVRLZWOQZCRWSWOCRZQWQWKWSWOCUDWTWPCWOWSWMWOWRWLVRWLWLVSCOZXBUE
      UFUGSWSWIXAWJEMUHZVSMPWSWIUIMMEUJXCULMMEUKUMWHMVSVREUNUOWJVSWLWLLZCCJZPZX
      AFXEVSCCUPUQWOXDQZCRZCRXHXFXAXHCXGCURVCXDCCVSUSWOXGCXDWOCVDUTSVAVBVEVFWOW
      ECVRWGWMWNWDVSWMWLVRWLVRWMVGZXIVHVIVJTTTVKWAVLWAWCLVTVMABWAVNVOABWDVPVQ
      $.
---------The proof of "fsplit" (732 bytes) ends above this line.
MM-PA>

Reply:
@avekens : your proof has to have a DV condition violation. Did you do `MM> VERIFY PROOF fsplit` ?

Reply:
@benjub Yes, you are right, I get:

```
There is a disjoint variable ($d) violation at proof step 160.  Assertion
"df-id" requires that variables "x" and "y" be disjoint.  But "x" was
substituted with "z" and "y" was substituted with "z".  These substitutions
have variable "z" in common.
```

I thought I had checked this, but maybe I mixed up something.

As a consequence: we should not change the proof of ~fsplit, and take it as an example for the use of ~dfid2 (when it is not easily replaceable by ~df-id).

Reply:
@avekens Ok.  Then I'm going to merge.